mmpose/core/evaluation/eval_hooks.py

```diff
@@ -32,15 +32,15 @@ class EvalHook(Hook):
 
     rule_map = {'greater': lambda x, y: x > y, 'less': lambda x, y: x < y}
     init_value_map = {'greater': -inf, 'less': inf}
-    greater_keys = ['acc', 'mAP', 'PCK', 'AUC']
-    less_keys = ['loss', 'EPE']
+    greater_keys = ['acc', 'ap', 'ar', 'pck', 'auc']
```

Comment:
why not add 'map' in this list

Reply:
If a key includes 'ap' (e.g., AP, AP 0.5, AP small), then it belongs to the group of greater_keys.

Reply:
i mean, can we avoid changing the 80 many files?